EleksTubeHAX_pio/src/TFTs.cpp

```diff
@@ -202,15 +220,20 @@ void TFTs::InvalidateImageInBuffer() { // force reload from Flash with new dimmi
 }
 
 void TFTs::ProcessUpdatedDimming() {
-  #ifdef DIM_WITH_ENABLE_PIN_PWM
+#ifdef DIM_WITH_ENABLE_PIN_PWM
+  //hardware dimming is done via PWM on the pin defined by TFT_ENABLE_PIN
+  //ONLY for IPSTUBE clocks in the moment! Other clocks may be damaged!
   if (enabled) {
     ledcWrite(TFT_PWM_CHANNEL, CALCDIMVALUE(dimming));
   } else {
+    //no dimming means 255 (full brightness)
```

Comment:
This is a little bit misleading, actually `CALCDIMVALUE(0)` calculates value for zero brightness.
255 is the result of inversion (because "active" Q1 is actually LED turned off).

Reply:
Got it! I changed the comments in another branch which I will merge soon...Still fighting with another problem...